Shell/UIHelper.cpp

```diff
@@ -115,7 +115,7 @@ bool szsOutputMode() {
   return (Lib::env.options && Lib::env.options->outputMode() == Shell::Options::Output::SZS);
 }
 
-ostream& addCommentSignForSZS(ostream& out)
+ostream& addCommentSignForSZS(std::ostream& out)
```

Comment:
`std::ostream` in return type too please!

Reply:
hunted for some more of these.